src/rgw/rgw_rest.h

```diff
@@ -385,6 +391,9 @@ class RGWHandler_REST : public RGWHandler {
   static int allocate_formatter(struct req_state *s, int default_formatter,
 				bool configurable);
 public:
+  static const int MAX_BUCKET_NAME_LEN = 255;
+  static const int MAX_OBJ_NAME_LEN = 1024;
```

Comment:
`constexpr` for those statics would be even more preferred than `const`. Someone in the future might want to make an arrays basing on them.